tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "allowJs": true,
-    "module": "esNext",
+    "module": "commonjs",
```

Comment:
I dont really understand what you want to accomplish here but I trust you that this is good :)

Reply:
It works this way but doesn't the other way *shrug*

Reply:
:+1: